src/Bundler/EntryPoint.php

```diff
@@ -52,58 +62,29 @@ public function getFile(): File
     }
 
     /**
-     * Return all non-vendor files.
+     * Returns the files to build
      *
-     * @return DependencyNodeInterface[]
+     * @return array
      */
-    public function getBundleFiles(): array
+    public function getFilesToBuild(string $output_dir): array
     {
-        return $this->bundle_files;
+        $result = [];
+        foreach ($this->files as $split_file => $dependencies) {
+            $result[$output_dir . '/' . $split_file] = $dependencies;
+        }
+        foreach ($this->getAssetFiles() as $asset_file) {
```

Comment:
you can use the property instead of the getter.

Reply:
in the updated pr the getter no longer exists